nuget/nunitlite.nuspec

```diff
@@ -37,10 +37,15 @@ How to use this package:
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
     <file src="bin/net-2.0/nunitlite.dll" target="lib/net20" />
+    <file src="bin/net-2.0/nunitlite-runner.exe" target="lib/net20" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
+    <file src="bin/net-4.0/nunitlite-runner.exe" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
+    <file src="bin/net-4.5/nunitlite-runner.exe" target="lib/net45" />
     <file src="bin/portable/nunitlite.dll" target="lib/dotnet" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/dotnet" />
     <file src="bin/portable/nunitlite.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
+    <file src="bin/portable/nunitlite-runner.exe" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
```

Comment:
I am pretty sure that this and the `lib/dotnet` one above are wrong. The runner is a .NET 4.5 console application and won't actually run on most of the platforms specified. It will only run on the desktop and is functionally equivalent to the net-4.5 runner, if not identical now that it doesn't include tests.

EXE's are not portable. If we want to go for this approach, we should be creating platform specific nunitlite-runners for each potential platform where it is possible and including them in platform specific lib directories. Of the targets listed there, .NET Core is probably the only one with a console that we can use. You can shell into Android to the console, but I am not sure how practical it would be, even in a CI scenario.

I don't think we need to solve this with this PR, it is getting too big already. I think you should remove those two nunitlite-runner.exe's from dotnet and portable and we can create another issue to create additional platform specific runners.